src/ffi.md

```diff
@@ -718,17 +718,116 @@ void register(int (*f)(int (*)(int), int)) {
 
 No `transmute` required!
 
-## FFI and panics
+## FFI and unwinding
 
-It’s important to be mindful of `panic!`s when working with FFI. A `panic!`
-across an FFI boundary is undefined behavior. If you’re writing code that may
-panic, you should run it in a closure with [`catch_unwind`]:
+It’s important to be mindful of unwinding when working with FFI. Each
+non-`Rust` ABI comes in two variants, one with `-unwind` and one without. If
```

Comment:
Maybe the below is clearer?

```suggestion
non-`Rust` ABI comes in two variants, one with `-unwind` suffix and one without. If
```

Reply:
Ooh, this is good, and much better than `appended`, which I used in my initial draft for the Reference updates. Thanks.